chat-components/src/components/outofofficehourspane/common/defaultProps/defaultOOOHPaneControlProps.ts

```diff
@@ -6,4 +6,5 @@ export const defaultOOOHPaneControlProps: IOOOHPaneControlProps = {
     hideOOOHPane: false,
     hideTitle: false,
     titleText: "Thanks for contacting us. You have reached us outside of our operating hours. An agent will respond when we open.",
+    openLinkInNewTab: false
```

Comment:
Please update defaultDummyProps.ts for this change

Reply:
Yes will update once chat-component changes go through will pull latest build with chat-widget and add this prop